rootfs/etc/haproxy/template/haproxy.tmpl

```diff
@@ -127,7 +127,7 @@ frontend httpfront
 ###### HTTPS frontend (tcp mode)
 ######
 frontend httpsfront
-    bind *:443
+    bind *:443{{ if $cfg.UseProxyProtocol }} accept-proxy{{ end }}
```

Comment:
Would it makes sense to have an two stats rules (one for external use, and one for internal use) so they can be configured independently? That way the external one can use `accept-proxy`, while the internal one can omit it for Kubernetes readiness/liveness checks.

Anyway, it sounds like that will be part of issue #21. Are you comfortable with the changes in this PR? (The comments to start.sh have been added)

Thanks for the feedback!